ui/js/directives/treeherder/clonejobs.js

```diff
@@ -818,8 +818,16 @@ treeherder.directive('thCloneJobs', [
 
         var isOnScreen = function(el){
             var viewport = {};
-            viewport.top = $(window).scrollTop() + $("#global-navbar-container").height();
-            viewport.bottom = viewport.top + $(window).height() - $("#info-panel").height();
+            viewport.top = $(window).scrollTop() + $("#global-navbar-container").height() + 20;
+            var filterbarheight = $(".active-filters-bar").height();
+            if (filterbarheight > 0) {
```

Comment:
could be:

    viewport.top = filterbarheight ? viewport.top + filterbarheight : viewport.top;